101-data-factory-v2-blob-to-blob-copy/azuredeploy.json

```diff
@@ -0,0 +1,186 @@
+{
+  "contentVersion": "1.0.0.0",
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "parameters": {
+    "dataFactoryLocation": {
```

Comment:
if the location is restricted to these two locations then put allowedValues on the param and put a note in the description

Reply:
Fixed.